pyvista/plotting/render_window_interactor.py

```diff
@@ -22,6 +22,40 @@
 _CLASSES = {}
 
 
+class Timer:
+    """Timer class.
+
+    Parameters
+    ----------
+    max_steps : int
+        Maximum number of steps for integrating a timer.
+
+    iren : RenderWindowInteractor
+        Rendering win and interactor.
+
+    callback : callable
+        A callable that takes one arguments (step argument is need).
+    """
+
+    def __init__(self, max_steps, iren, callback):
+        """Initialize."""
+        self.step = 0
+        self.max_steps = max_steps
+        self.iren = iren
```

Comment:
This doesn't seem to be used within this class. Also, including a reference to the render window interactor might mean that the render window interactor doesn't get collected during garbage collection.

Reply:
I'm sorry. I did not understand it well due to my lack of knowledge. In my mind, `iren` is used in the execute method. If you don't mind, could you suggest a change?

Reply:
It's embarrassing! I reviewed the code for the first time in a week and realized my misunderstanding. I will delete this line.
```suggestion
```